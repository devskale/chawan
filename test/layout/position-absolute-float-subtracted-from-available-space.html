<!DOCTYPE html>
<body>
<div style="position: relative; width: 30ch; height: 5em; background: purple">
<div style="display: inline-block; float: left; width: 50%; background: red">
asdf
</div>
<div style="display: inline-block; float: left; position: absolute">
test test test test test test test test test test test test test test
