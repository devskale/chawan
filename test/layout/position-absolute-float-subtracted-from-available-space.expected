[48;2;255;0;0masdf           [48;2;128;0;128mtest test test [49m
[48;2;128;0;128m               test test test [49m
[48;2;128;0;128m               test test test [49m
[48;2;128;0;128m               test test test [49m
[48;2;128;0;128m               test test      [49m
